git-version-control/gitignore.md

```diff
@@ -118,8 +118,10 @@ You should no longer see *.my-secret-keys* in your untracked files, because git
 
 ### Check for Understanding
 
-- Small group discussion:
+Form small groups and discuss:
 
   - What sorts of files should you add to your .gitignore?
-
+  - How can you add to .gitignore each file of one type?
```

Comment:
Please change to: how can you add all files of one type to your .gitignore file?

Reply:
the objectives they should be making sure they understand are: 
- [x] Adding unwanted files to their .gitignore
- [ ] Know what data to protect from public view
- [ ] Know which files don't need to be committed